hashing/te-tag-query-java/com/facebook/threatexchange/TETagQuery.java

```diff
@@ -334,20 +328,6 @@ public void handle(
           pageSize = Integer.valueOf(args[0]);
           args = Arrays.copyOfRange(args, 1, args.length);
 
-        } else if (option.equals("--indicator-type")) {
```

Comment:
Should we make "--indicator-type" a constant?

Reply:
we could? all the flags are literals & we could constantize them? personally i prefer to constantize things which appear more often in a method; "rule of three" -- ? :)